xclim/data/fr.json

```diff
@@ -141,330 +141,283 @@
-    "comment": "",
     "title": "Durée totale des vagues de chaleur",
     "abstract": "Durée totale des vagues de chaleur. Une vague de chaleur se produit lorsque les températures journalières minimales et maximales excèdent des seuils donnés durant un certain nombre de jours."
   },
   "HEAT_WAVE_MAX_LENGTH": {
     "long_name": "Longueur maximale des vagues de chaleur (Tmin > {thresh_tasmin} et Tmax > {thresh_tasmax} pour {window} jours au moins)",
     "description": "Longueur maximale {freq:f} des vagues de chaleur. Une vague de chaleur se produit lorsque les températures quotidiennes minimales et maximales excèdent {thresh_tasmin} et {thresh_tasmax}, respectivement, durant au moins {window} jours.",
-    "comment": "",
     "title": "Longueur maximale des vagues de chaleur",
     "abstract": "Longueur maximale des vagues de chaleur. Une vague de chaleur se produit lorsque les températures journalières minimales et maximales excèdent des seuils donnés durant un certain nombre de jours."
   },
   "HEAT_WAVE_INDEX": {
     "long_name": "Nombre de jours de vague de chaleur",
     "description": "Nombre {freq:m} de jours de vague de chaleur, définie comme cinq jours consécutifs ou plus où la température est au-dessus de {thresh}.",
     "title": "Indice de vague de chaleur",
-    "comment": "",
     "abstract": "Nombre de jours de vagues de chaleur. Une vague de chaleur se produit lorsque la température journalière maximale excède un seuil durant au moins 5 jours."
   },
   "TG": {
     "long_name": "température journalière moyenne",
     "description": "température journalière moyenne estimée par la médiane des températures maximales et minimales.",
     "title": "Moyenne des températures maximales et minimales",
-    "comment": "",
     "abstract": "La température journalière moyenne en assumant une distribution symétrique des températures.  Tg = (Tx + Tn) / 2"
   },
   "TG_MIN": {
     "long_name": "Minimum de la température journalière moyenne",
     "description": "Minimum {freq:m} de la température journalière moyenne.",
     "title": "Minimum de la température journalière moyenne",
-    "comment": "",
     "abstract": "Minimum de la température journalière moyenne."
   },
   "TG_MAX": {
     "long_name": "Maximum de la température journalière moyenne",
     "description": "Maximum {freq:m} de la température journalière moyenne.",
     "title": "Maximum de la température journalière moyenne",
-    "comment": "",
     "abstract": "Maximum de la température journalière moyenne."
   },
   "TG_MEAN": {
-    "long_name": "Moyenne de la température journalière moyenne",
-    "description": "Moyenne {freq:f} de la température journalière moyenne.",
-    "title": "Moyenne de la température journalière moyenne",
-    "comment": "",
-    "abstract": "Moyenne de la température journalière moyenne."
+    "long_name": "Moyenne de la température journalière",
```

Comment:
Is there a reason sometimes it is "quoditienne" et sometimes "journalière" ?

Reply:
From what I understand, it has something to do with the context and the repetitive nature of the event. I won't pretend to understand the intricacies, but journalière appears to be the favoured word.

More on the differences: https://french.stackexchange.com/questions/34168/quotidiennement-ou-journalier#34169

Reply:
Intéressante source @Zeitsperre !
Donc la température serait journalière, parce qu'on précise ici le pas de temps d'une variable (qui varie).

Et boboy, il faudrait donc changer ça partout pour harmoniser.

Reply:
ah bon! Je ne connaissais pas cette subtilité. Je pense qu'il faudrait mettre journalière partout alors ( pour précipitation aussi)!